src/routes/token.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-underscore-dangle */
 /* eslint-disable camelcase */
+require('dotenv');
```

Comment:
This will possibly crash in production: `dotenv` is a dev dependency, but you include it in a file that runs in prod.

You probably want to require `dotenv` in [test/helpers.js](./test/helpers.js)?

Reply:
ok understood